### ISSUE TITLE
SpeedDialChild's child overwhelmed

### ISSUE THREAD

**Issue by FTholin** [darioielardi/flutter_speed_dial#57]
Hi, 

First of all, I would like to thank you to allow me to use your awesome widget.

I contact you because I encounter an uncomfortable situation and I would like to know if alternatives exist or if I do something wrong.

I use SpeedDialChild with Image.asset and my image overwhelm the button. I also try to use width and height property, or IconButton but without success.

<img width="46" alt="Screen Shot 2020-02-24 at 12 42 57 PM" src="https://user-images.githubusercontent.com/39242929/75151121-3216d980-5706-11ea-824c-6c623edc7118.png">

I would be pleased if you are more infos to give me.

Best regards.

**Comment by darioielardi** [comment]
If you provide a minimum reproduction repo I'd be happy to help you!

**Comment by FTholin** [comment]
```dart
import 'package:flutter/material.dart';
import 'package:flutter_speed_dial/flutter_speed_dial.dart';

void main() => runApp(MyApp());

class MyApp extends StatefulWidget {
  @override
  State<StatefulWidget> createState() {
    return _MyAppState();
  }
}

class _MyAppState extends State<MyApp> {

  bool addButtonSelected = false;
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      home: Scaffold(
        appBar: AppBar(
          title: Text('Welcome to Flutter'),
        ),
        body: Center(
          child: Text('I would like the second button be center like with icon and not overwhelmed.'),
        ),
        floatingActionButton: SpeedDial(
          visible: true,
          child: new IconTheme(
            data: new IconThemeData(color: Colors.white),
            child: new Icon(addButtonSelected ? Icons.clear : Icons.add),
          ),
          animatedIconTheme: IconThemeData(size: 22.0),
          onOpen: () {
            setState(() {
              addButtonSelected = true;
            });
          },
          onClose: () {
            setState(() {
              addButtonSelected = false;
            });
          },
          curve: Curves.bounceIn,
          children: [

            SpeedDialChild(
              child: Icon(Icons.nature, color: Colors.white,),
              onTap: () {},
              label: 'Cool with Icon',
              labelStyle: TextStyle(fontWeight: FontWeight.w500, fontSize: 20, color: Colors.white),
              labelBackgroundColor: Colors.blue,
            ),
            SpeedDialChild(
              // Load local image
              child: Image.asset("group.png", height: 10, width: 10,),
              onTap: () {},
              label: 'With Image.asset as child',
              labelStyle: TextStyle(fontWeight: FontWeight.w500, fontSize: 20, color: Colors.white),
              labelBackgroundColor: Colors.blue,
            ),
          ],
        ),
      ),
    );
  }
}
```

**Comment by prateekmedia** [comment]
Looks like it's fixed in #67  as I have merged it in #90, if it still persists ask me to reopen this.

**Comment by prateekmedia** [comment]
Just wrap your Image widget with a Center widget.
